draft-ietf-taps-arch.md

```diff
@@ -448,33 +448,34 @@ The following categories of events can be delivered to an application:
 
-* Abort: The action the application takes on a Connection to indicate a Close and also indicate that the Transport Services system should not attempt to deliver any outstanding data, and immediately drop the connection. This is intended for immediate, usually abnormal, termination of a connection.
+* Abort: The action the application takes on a Connection to indicate a Close and also indicate that the Transport System should not attempt to deliver any outstanding data, and immediately drop the connection. This is intended for immediate, usually abnormal, termination of a connection.
 
 ### Connection Groups
 
-A Connection Group is a set of Connections that shares properties and caches. A Group represents state for managing Connections within a single application, and does not require end-to-end protocol signaling. For multiplexing transport protocols, only Connections within the same Connection Group are allowed to be multiplexed together.
+A Connection Group is a set of Connections that shares properties and caches. A Connection Group represents state for managing Connections within a single application, and does not require end-to-end protocol signaling. For multiplexing transport protocols, only Connections within the same Connection Group are allowed to be multiplexed together.
 
 When the API clones an existing Connection, this adds a new Connection to the Connection Group. A change to one of the Connection Properties on any Connection in the Connection Group automatically changes the Connection Property for all others. All Connections in a Connection Group share the same set of Connection Properties except for the Connection Priority. These Connection Properties are said to be entangled. 
 
 For multiplexing transport protocols, only Connections within the same Connection Group are allowed to be multiplexed together. Passive Connections can also be added to a Connection Group, e.g., when a Listener receives a new Connection that is just a new stream of an already active multi-streaming protocol
 instance.
 
-While Connection Groups are managed by the transport system, an application can define Connection Contexts to control caching boundaries, as discussed in {{conn-context}}.
+While Connection Groups are managed by the Transport System, an application can define Connection Contexts to control caching boundaries, as discussed in {{conn-context}}.
 
-## Transport Services Implementation Concepts
+## Transport System Implementation
 
-This section defines the set of objects used internally to a system or library to implement the functionality needed to provide a transport service across a network, as required by the abstract interface.
+This section defines the key concepts of the Transport System component of the Transport Services Architecture.
```

Comment:
This gets odd. If the Transport System is not a complete instantiation of the architecture (as I thought up to here), but just a component, then how can it be one of the "key concepts" of itself?  because it's also the next item in the list.

Reply:
Made the Transport System more clearly sit below the API when first introduced earlier (subtle because the "API" is only realised when you have a system designed to implement it, but at least tries to make reading less painful. We didn't talk about a "component" here, so suggest we don't. Text updated.